src/main.py

```diff
@@ -10,36 +10,75 @@
 
 Authors: Zein Youssef and Steven Dropiewski
 """
-import os
-
 from follower import Follower
 from leader import Leader
 from tagrec import TagRecognition
 
+import os
+import subprocess
+import time
+
 
 """
-Determines if a Bluetooth controller is connected. 
+Determines if a Bluetooth controller is connected.
 
 Returns true if a device is connected and false if not.
 """
 def is_controller_connected():
     return os.path.exists('/dev/input/js0')
 
 
+"""
+This function calls the ./bt_toggle script with the specified
+command and address.
+"""
+def bt_toggle(command, address):
+    subprocess.call(['./bt_toggle.sh', command, address])
+
+
 def main():
     tag = TagRecognition()
+    bl = Bluetoothctl()
+    BT_ADDRESS = '5C:BA:37:26:6D:9A'
 
-    # If an ARTag is detected, there is a vehicle ahead of this vehicle so it will follow.
-    # If an ARTag is not detected, no vehicle is present so this vehicle will be a leader.
+    # If an ARTag is detected, the vehicle will disconnect from a Bluetooth
```

Comment:
I'd leave this unchanged. I think saying that it becomes a leader or a follower (like it did) is more detailed. Otherwise, this can mean that if the leader sees an ARTag, it disconnects from Bluetooth but stays a leader. I think it was better before.